src/Makefile.am

```diff
@@ -156,25 +156,20 @@ eos_updater_SOURCES = \
 
 dist_man8_MANS += docs/eos-updater.8
 
+eos_updater_dbus_if := com.endlessm.Updater
+
 dbusconfdir = $(datadir)/dbus-1/system.d
 dbussystemservicedir = $(datadir)/dbus-1/system-services
 
-dbusconf_DATA = $(eos_updater_dbus_if).conf
-dbussystemservice_DATA = $(eos_updater_dbus_if).service
+dbusconf_DATA = $(srcdir)/$(eos_updater_dbus_if).conf
+dbussystemservice_DATA = $(srcdir)/$(eos_updater_dbus_if).service
```

Comment:
Hm hm hm, `make distcheck` passed here, I did run it before filing this PR. I think these things are distributed automatically if they are not built and installed somewhere, but that sounds overly smart/magic, so I'll check it.